src/app/personside/infotabs/meldinger/traadliste/TraadListeElement.tsx

```diff
@@ -1,25 +1,36 @@
 import * as React from 'react';
-import { Traad } from '../../../../../models/meldinger/meldinger';
+import { Meldingstype, Traad } from '../../../../../models/meldinger/meldinger';
 import VisMerKnapp from '../../../../../components/VisMerKnapp';
 import Element from 'nav-frontend-typografi/lib/element';
 import styled from 'styled-components';
 import { meldingstypeTekst, temagruppeTekst } from '../utils/meldingstekster';
 import Normaltekst from 'nav-frontend-typografi/lib/normaltekst';
 import { theme } from '../../../../../styles/personOversiktTheme';
-import SakIkkeTilgangIkon from '../../../../../svg/SakIkkeTilgangIkon';
 import { formatterDatoTid } from '../../../../../utils/dateUtils';
-import { sisteSendteMelding } from '../utils/meldingerUtils';
+import { erMonolog, sisteSendteMelding } from '../utils/meldingerUtils';
+import Oppmote from '../../../../../svg/Oppmote';
```

Comment:
Tror jeg ville kalt alle disse for `...Ikon` eller noe lignende. (`OppmoteIkon, TelefonIkon, osv...`)